Main_Project/src/Views/ManageInventoryGUI.java

```diff
@@ -256,6 +256,7 @@ else if(e.getSource() == logoutButton){
             tablePanel.remove(reloadButton);
             tablePanel.repaint();
             tablePanel.add(loadButton);
+            linkDB.closeConnection();
```

Comment:
Nice add here! We should go back and add this for all logout button instances in out GUI pages. Should be done in a different PR though.